spec/features/quotes_spec.rb

```diff
@@ -1,12 +1,16 @@
 require 'rails_helper'
 require 'capybara/rails'
+require 'action_view'
+require 'action_view/helpers'
+require 'date'
```

Comment:
Halah says: "Date is for Date.today used below, and action_view/helpers was in a test that I decided to remove. I'll remove those :) "